src/main/java/org/codefx/junit/io/extension/DisabledOnOs.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2015-2016 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.codefx.junit.io.extension;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+
+/**
+ * {@code @DisabledOnOs} is used to signal that the annotated test class or
+ * test method is <em>disabled</em> on the specified operating systems.
+ *
+ * <p>When applied at the class level, all test methods within that class
+ * are automatically disabled as well if on the specified operating system.
+ *
+ * @since 1.0
+ */
+@Target({ ElementType.TYPE, ElementType.METHOD })
+@Retention(RetentionPolicy.RUNTIME)
+@ExtendWith(OsCondition.class)
+public @interface DisabledOnOs {
+
+	/**
+	 * Operating systems on which the test or test container is disabled.
+	 */
+	OS[] value() default {};
```

Comment:
Don't provide a default value for an annotation attribute if its required.

In other words, declaring `@DisabledOnOs` without specifying and OS is pointless. 😉 

Same for `@EnabledOnOs`.